components/docusign/sources/common.js

```diff
@@ -0,0 +1,43 @@
+const docusign = require("../docusign.app.js");
+
+module.exports = {
+  props: {
+    docusign,
+    account: {
+      propDefinition: [
+        docusign,
+        "account",
+      ],
+    },
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+  methods: {
+    _getBaseUri() {
+      return this.db.get("baseUri");
+    },
+    _setBaseUri(baseUri) {
+      this.db.set("baseUri", baseUri);
+    },
+    monthAgo() {
```

Comment:
Since this is only used in components/docusign/sources/envelope-sent-or-complete/envelope-sent-or-complete.js, recommend moving it to that file. When that is done, there is really no compelling reason for a common.js file, and I recommend this file be deleted.

Reply:
Moved monthAgo() to envelope-sent-or-complete.js and removed common.js file